wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -904,7 +904,7 @@ public int onStartCommand(final Intent intent, final int flags, final int startI
             } else if(BlockchainService.ACTION_RESET_BLOOMFILTERS.equals(action)) {
                 if (peerGroup != null) {
                     log.info("recalculating bloom filters");
-                    peerGroup.recalculateFastCatchupAndFilter(PeerGroup.FilterRecalculateMode.SEND_IF_CHANGED);
+                    peerGroup.recalculateFastCatchupAndFilter(PeerGroup.FilterRecalculateMode.FORCE_SEND_FOR_REFRESH);
```

Comment:
Not sure if FORCE_SEND_FOR_REFRESH is required vs SEND_IF_CHANGED, but that is how most of the testing was done.